Sources/Commands/SwiftPackageTool.swift

```diff
@@ -412,7 +437,7 @@ public enum PackageMode: String, StringEnumArgument {
     case edit
     case fetch
     case generateXcodeproj = "generate-xcodeproj"
-    case generateCompletionScript = "generate-completion-script"
+    case completionTool = "completion-tool"
```

Comment:
I've named it `completion-tool` instead of `shell-autocomplete` as it is technically not autocompleting. Only suggestion completions when asked.